packages/tasks/src/model-libraries.ts

```diff
@@ -419,6 +419,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		repoUrl: "https://github.com/DigitalPhonetics/IMS-Toucan",
 		countDownloads: `path:"embedding_gan.pt" OR path:"Vocoder.pt" OR path:"ToucanTTS.pt"`,
 	},
+	index_tts: {
```

Comment:
```suggestion
	index-tts: {
```
(more consistent with other tags)

Can you add `library_name: index-tts` to https://huggingface.co/IndexTeam/Index-TTS model card metadata so that the model is correctly listed under https://huggingface.co/models?other=index-tts. We usually merge these PRs only when at least 1 model is listed.